src/entities/Badges/utils.ts

```diff
@@ -1,9 +1,12 @@
 import { Contract } from '@ethersproject/contracts'
-import { JsonRpcProvider } from '@ethersproject/providers'
 import { abi as BadgesAbi } from '@otterspace-xyz/contracts/out/Badges.sol/Badges.json'
+import logger from 'decentraland-gatsby/dist/entities/Development/logger'
 import { ethers } from 'ethers'
 
 import RpcService from '../../services/RpcService'
+import { OTTERSPACE_DAO_RAFT_ID } from '../Snapshot/constants'
+
+import { GAS_MULTIPLIER, GasConfig } from './types'
 
 const RAFT_OWNER_PK = process.env.RAFT_OWNER_PK || ''
```

Comment:
Are we sure that this pk cannot be seen from the front?

Reply:
I think that for that to happen the PK should be in `prd.json`